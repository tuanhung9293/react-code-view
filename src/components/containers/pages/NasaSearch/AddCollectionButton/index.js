import React from 'react'
import PrimaryButton from 'components/ui/buttons/PrimaryButton'

const AddCollectionButton = ({itemData}) => {
  return (
    <PrimaryButton outlined wide to={`/nasa-search/add-media/${itemData.nasaId}`}>
      Add to NASA Collection
    </PrimaryButton>
  )
}

export default AddCollectionButton
