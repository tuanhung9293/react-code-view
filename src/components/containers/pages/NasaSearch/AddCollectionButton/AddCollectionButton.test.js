import React from 'react'
import {shallow} from 'enzyme'
import AddCollectionButton from './'

const itemData = {
  nasaId: 'nasaId'
}

describe('renders', () => {
  it('renders AddCollectionButton', () => {
    expect(shallow(
      <AddCollectionButton itemData={itemData} />
    )).toMatchSnapshot()
  })
})

