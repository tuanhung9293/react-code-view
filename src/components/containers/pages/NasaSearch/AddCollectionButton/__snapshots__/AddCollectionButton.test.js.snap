// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders renders AddCollectionButton 1`] = `
<PrimaryButton
  outlined={true}
  to="/nasa-search/add-media/nasaId"
  wide={true}
>
  Add to NASA Collection
</PrimaryButton>
`;
